include/gmapping/grid/array2d.h

```diff
@@ -7,6 +7,10 @@
 
 #include <iostream>
 
+#ifdef _WIN32
+#define __PRETTY_FUNCTION__ __FILE__":" << __LINE__
```

Comment:
Yeah, I guess you don't have `func` in that case. Either way, I think it should include the function name in there.